cicd.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+#git submodule update --init --recursive
```

Comment:
remove unused line (or add explanation why this should be kept)

Reply:
removed